src/config/mod.rs

```diff
@@ -308,21 +337,37 @@ pub(crate) fn generate() -> Config {
         }
     });
 
-    let mut release_steps = if github.is_some() || gitea.is_some() {
+    Config {
+        workflows: generate_workflows(github.is_some() || gitea.is_some(), &packages),
+        jira: None,
+        github,
+        gitea,
+        packages,
+    }
+}
+
+fn generate_workflows(has_forge: bool, packages: &[Package]) -> Vec<Workflow> {
+    let (commit_message, variables) = if packages.len() < 2 {
+        let mut variables = IndexMap::new();
+        variables.insert(String::from("$version"), Variable::Version);
```

Comment:
Are you suggesting that we default to replacing the string `$version` with the version variable, even if that variable wasn't declared? That would be tricky because someone might want that literal string and now doesn't have an obvious way to opt out. Or maybe I'm misunderstanding...

Reply:
mmh we could give the user a simple flag `defaults` field on `variables::Template` they can set to false per step where default behavior isn't wished and no other variables are set, if all they care for is the literal `$version`. hat way they can override or disable the default behavior. Additionally i would only default to this if no variables are configured at all.
 
 At least from my perspective when using it, i pretty much exclusively used the variables as described in the docs. Mainly because i think it's a sane default ^^